pkg/apis/volumesnapshot/v1alpha1/types.go

```diff
@@ -241,3 +249,15 @@ type CSIVolumeSnapshotSource struct {
 	// +optional
 	RestoreSize *int64 `json:"restoreSize,omitempty" protobuf:"bytes,4,opt,name=restoreSize"`
 }
+
+// DeletionPolicy describes a policy for end-of-life maintenance of volume snapshot contents
+type DeletionPolicy string
+
+const (
+	// VolumeSnapshotContentDelete means the snapshot content will be deleted from Kubernetes on release from its volume snapshot.
+	// The volume plugin must support Deletion.
+	VolumeSnapshotContentDelete DeletionPolicy = "Delete"
+	// VolumeSnapshotContentRetain means the volume will be left in its current phase (Released) for manual reclamation by the administrator.
+	// The default policy is Retain.
```

Comment:
update the comment here since the default policy `Delete`

Reply:
Sorry actually here we should set default policy for VolumeSnapshotContent is Retain.
Similar to Persistent Volumes, if VolumeSnapshotContent is created manually, the default set is Retain. If VolumeSnapshotContent is dynamically created through VolumeSnapshot, the default policy is Delete which is set through VolumeSnaphshotClass.